core/src/main/java/binnie/core/block/TileEntityMetadata.java

```diff
@@ -40,8 +40,10 @@ public static ItemStack getItemStack(final Block block, final int damage) {
 		return itemStack;
 	}
 
+	private static final int MAX_ITEM_DAMAGE = 16387;
```

Comment:
Any idea what the significance of this number is?
I thought it might be a power of 2 minus 1, but the nearest power of 2 is 2^14 (16384) which is also a weird number.

Reply:
No idea, I was trying to find this value (binnie, forestry, forge) but no successed

Reply:
maybe power of 2 + wear points on a frame? thought frames had more points than that though https://github.com/ForestryMC/Binnie/blob/da2df5b5985ebf3353577857bda2733c84397901/extrabees/src/main/java/binnie/extrabees/items/ItemHiveFrame.java#L77.